src/plugins/discover/server/ui_settings.ts

```diff
@@ -199,4 +200,15 @@ export const getUiSettings: () => Record<string, UiSettingsParams> = () => ({
     category: ['discover'],
     schema: schema.boolean(),
   },
+  [SHOW_MULTIFIELDS]: {
+    name: i18n.translate('discover.advancedSettings.discover.showMultifields', {
+      defaultMessage: 'Show multi fields',
+    }),
+    description: i18n.translate('discover.advancedSettings.discover.showMultifieldsDescription', {
+      defaultMessage: `When enabled will show multi fields in document viewer. In majority of cases, multi-fields are the same as regular fields. This option is only available when searchFieldsFromSource is off. `,
+    }),
```

Comment:
FYI dear @gchaps a new config flag and wording

Reply:
Adding a screenshot to make things more clear:
![Uploading Screenshot 2021-06-16 at 12.01.11.png…]()

Reply:
Here is a suggestion

**Show multi-fields**

Controls whether [multi-fields](https://www.elastic.co/guide/en/elasticsearch/reference/current/multi-fields.html) display in the document table. Available when `searchFieldsFromSource` is off.

Notes:
- multi-fields has a hyphen
- Not sure what the document viewer is.  Does it refer to the document table?
- Is the second sentence needed? Not sure what "regular field" means.

Reply:
Document viewer is the part in the expanded view of the document table that shows all fields in the document:
<img width="747" alt="Screenshot 2021-06-16 at 15 44 12" src="https://user-images.githubusercontent.com/1937956/122240786-f1723f00-ceb9-11eb-8c61-680b70d68683.png">


As for the second sentence, I am open to suggestions on how to improve it. I would like to keep the emphasis on the fact that in 99% of cases, there should be no difference between an original field value and its multi-field counterpart.

Reply:
**Show multi-fields**

Controls whether [multi-fields](https://www.elastic.co/guide/en/elasticsearch/reference/current/multi-fields.html) display in the expanded document view. In most cases, multi-fields are the same as the original field.  This option is only available when `searchFieldsFromSource` is off.